bootstrap4/.js/src/main/scala/io/udash/bootstrap/alert/DismissibleUdashAlert.scala

```diff
@@ -1,17 +1,17 @@
 package io.udash.bootstrap
 package alert
 
-import io.udash._
+import io.udash.*
 import io.udash.bindings.modifiers.Binding
 import io.udash.bootstrap.button.{UdashButton, UdashButtonOptions}
 import io.udash.bootstrap.utils.{BootstrapStyles, BootstrapTags}
 import org.scalajs.dom.Element
-import scalatags.JsDom.all._
+import scalatags.JsDom.all.*
+import com.avsystem.commons.SharedExtensions.universalOps
```

Comment:
Why `universalOps` has to be imported manually?

Reply:
```
Implicit method universalOps was found in a package prefix of the required type, which is not part of the implicit scope in Scala 3.
For migration, add `import com.avsystem.commons.SharedExtensions.universalOps`.
```